code/modules/clothing/spacesuits/rig/modules/ds13/healthbar.dm

```diff
@@ -6,16 +6,14 @@
 	icon_state = "healthbar"
 	use_power_cost = 0
 	origin_tech = list(TECH_MAGNET = 3, TECH_BIO = 3, TECH_ENGINEERING = 5)
-	suit_overlay_inactive = "healthbar_100"
-	suit_overlay_active = "healthbar_100"
-	suit_overlay_used = "healthbar_100"
-	suit_overlay = "healthbar_100"
+	suit_overlay = "healthbar"
+	second_overlay =  "healthbar_100"
 	var/mob/living/carbon/human/user
 	process_with_rig = FALSE
 
-	suit_overlay_layer = EYE_GLOW_LAYER
```

Comment:
these changes need explanation

Reply:
If we use just an overlay layer for stasis it'll glow in dark places (including gun itself) so I decided to separate them